### PR TITLE
Library App: Using a camera

### PR DESCRIPTION
**Pull request by bhoomika08**

### CODE REVIEW

File: LibraryApp/src/components/navigators.js [hunk 1]
```diff
@@ -32,6 +33,7 @@ const Navigators = () => {
         />
         <Stack.Screen name={libraryForm} component={Form} />
         <Stack.Screen name={showBookDetails} component={ShowBook} />
+        <Stack.Screen name={scanQRCode} component={ScanCode} options={{title: 'SCAN QR CODE'}}/>
```
Comment (sawangupta92) [inline LibraryApp/src/components/navigators.js]: can we open this in modal? using react-navigation only

File: LibraryApp/src/components/shared/image-picker.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      } catch (err) {
+        console.warn(err);
+        return false;
+      }
+    } else return true;
+  };
+
+  // ------------ request Write Permission to Store Captured Image to Device --------- //
+  requestExternalWritePermission = async () => {
+    if (Platform.OS === 'android') {
```
Comment (sawangupta92) [inline LibraryApp/src/components/shared/image-picker.js]: what about iOS permission? can we manage that as well?

File: LibraryApp/src/components/shared/image-picker.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+      cameraType: 'back',
+      allowsEditing: true,
+      maxWidth: 300,
+      maxHeight: 400,
+      saveToPhotos: true,
+    };
+    let isCameraPermitted = await this.requestCameraPermission();
+    let isStoragePermitted = await this.requestExternalWritePermission();
+    if (isCameraPermitted && isStoragePermitted) {
+      launchCamera(options, (res) => {
```
Comment (sawangupta92) [inline LibraryApp/src/components/shared/image-picker.js]: this method is too big, can we break it?

File: LibraryApp/src/components/shared/image-picker.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+  // -------- Launch Device Gallery -------------- //
+  imageGalleryLaunch = () => {
+    const {onImageSelect} = this.props;
+    let options = {
+      maxWidth: 300,
+      maxHeight: 400,
+    };
+
+    launchImageLibrary(options, (res) => {
```
Comment (sawangupta92) [inline LibraryApp/src/components/shared/image-picker.js]: do we need to check permission here

File: LibraryApp/src/components/navigators.js (hunk 1 above)
Comment (bhoomika08) [reply LibraryApp/src/components/navigators.js]: Done

File: LibraryApp/src/components/shared/image-picker.js (hunk 3 above)
Comment (bhoomika08) [reply LibraryApp/src/components/shared/image-picker.js]: Done

File: LibraryApp/src/components/shared/image-picker.js (hunk 2 above)
Comment (bhoomika08) [reply LibraryApp/src/components/shared/image-picker.js]: Added ios permissions as well.

**Pull request merged**